pyupgrade/_plugins/typing_classes.py

```diff
@@ -38,7 +38,7 @@ def _unparse(node: ast.expr) -> str:
         else:
             slice_s = _unparse(node_slice)
         return f'{_unparse(node.value)}[{slice_s}]'
-    elif isinstance(node, ast.Str):
+    elif isinstance(node, ast.Str) or isinstance(node, ast.Bytes):
```

Comment:
`isinstance(x, A) or isinstance(x, B)` is better written as `isinstance(x, (A, B))`